packages/tracing/test/hub.test.ts

```diff
@@ -32,6 +32,16 @@ describe('Hub', () => {
       expect(hub.getScope()?.getTransaction()).toBe(transaction);
     });
 
+    it('should find an unsampled transaction which has been set on the scope', () => {
```

Comment:
[L]

The `sampled` attribute has 3 states: undefined, false and true. I understand we're using "sampled" to mean "sampled: true", and now "unsampled" to mean "sampled: false" -- what word are we using for the undefined case?

Unsampled sounds ambiguous to me, because "undecided" doesn't mean "decided not to do", but simply has not made a decision -- and "unsampled" could be "has not made a sampling decision"? Might be just me :/

Reply:
No, it's a fair point. It's how we've been using it, but you're right that it's open to misinterpretation. I'll fix it here, but we might think about looking at all the other places we talk about it that way, too.

Reply:
FWIW, W3C's [trace-context documentation](https://www.w3.org/TR/trace-context/#examples-of-http-traceparent-headers) uses "sampled" and "not sampled".

(I was reading that page as part of updating our docs)